airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/message/DestinationMessage.kt

```diff
@@ -251,6 +253,7 @@ data class EnrichedDestinationRecordAirbyteValue(
      */
     val meta: Meta?,
     val serializedSizeBytes: Long = 0L,
+    private val extractedAtAsTimestampWithTimezone: Boolean = false
```

Comment:
it feels weird that in GCS mode we set this to true, but leave it as false in standard inserts mode - is the plan to fix this when we refactor out the shared code?